src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs

```diff
@@ -13,84 +13,27 @@ namespace Microsoft.Tools.ServiceModel.Svcutil
 {
     internal class TargetFrameworkHelper
     {
-        public static ReadOnlyDictionary<Version, Version> NetStandardToNetCoreVersionMap { get; } = new ReadOnlyDictionary<Version, Version>(new SortedDictionary<Version, Version>
-         {
-            // Service Model requires netstandard1.3 so it is the minimum version that will work.
-             {new Version("1.3"), new Version("1.0") },
-             {new Version("1.4"), new Version("1.0") },
-             {new Version("1.5"), new Version("1.0") },
-             {new Version("1.6"), new Version("1.0") },
-             {new Version("1.6.1"), new Version("1.1") },
-             {new Version("2.0"), new Version("2.0") }
-         });
-
-        internal static SortedDictionary<Version, List<ProjectDependency>> NetCoreVersionReferenceTable = new SortedDictionary<Version, List<ProjectDependency>>
+        private static readonly Version s_minEOFNetVersion = new Version("5.0");
+        private static readonly Version s_maxEOFNetVersion = new Version("7.0");
+        internal static readonly List<string> s_supportedVersions = new List<string>() { "8.0", "9.0", "10.0" };
```

Comment:
suggest to use currentSupportedVersions.

Reply:
Updated.